src/divide.js

```diff
@@ -17,9 +17,6 @@ var _curry2 = require('./internal/_curry2');
  *
  *      R.divide(71, 100); //=> 0.71
  *
- *      var half = R.divide(R.__, 2);
- *      half(42); //=> 21
- *
```

Comment:
This example still lives, I moved it to `partial`. But we can move it back here if that is preferred.